libs/designsystem/src/lib/components/checkbox/checkbox.component.scss

```diff
@@ -1,52 +1,102 @@
 @import '../../scss/utils';
 
+$border-radius: 6px;
+$checkbox-icon-size: size('m');
+
 :host {
-  display: inline-block;
-  vertical-align: middle;
-}
+  display: block;
```

Comment:
Should this still be inline-block for checkboxes without a label and block when wrapped in label?